ui/component/fileRender/view.jsx

```diff
@@ -1,28 +1,24 @@
 // @flow
 import { URL } from 'config';
 import { remote } from 'electron';
-import React, { Suspense, Fragment } from 'react';
+import React, { Suspense } from 'react';
 import classnames from 'classnames';
-import LoadingScreen from 'component/common/loading-screen';
+import * as RENDER_MODES from 'constants/file_render_modes';
 import VideoViewer from 'component/viewers/videoViewer';
 import ImageViewer from 'component/viewers/imageViewer';
 import AppViewer from 'component/viewers/appViewer';
 import Button from 'component/button';
 import { withRouter } from 'react-router-dom';
 import AutoplayCountdown from 'component/autoplayCountdown';
 import { formatLbryUrlForWeb } from 'util/url';
-// @if TARGET='web'
-import { generateStreamUrl } from 'util/lbrytv';
```

Comment:
moved into proxied selector, this way we can assume when selecting a streamUrl it will just work